File: Entities/PedidoDetalle.cs
namespace Entidades
{
    using System;

    public class PedidoDetalle
    {
        private int _varIdPedido;
        private int _varIdProducto;
        private double _varCantidad;
        private double _varCantidadRecibida;

        public int IdPedido
        {
            get => 
                this._varIdPedido;
            set => 
                this._varIdPedido = value;
        }

        public int IdProducto
        {
            get => 
                this._varIdProducto;
            set => 
                this._varIdProducto = value;
        }

        public double Cantidad
        {
            get => 
                this._varCantidad;
            set => 
                this._varCantidad = value;
        }

        public double CantidadRecibida
        {
            get => 
                this._varCantidadRecibida;
            set => 
                this._varCantidadRecibida = value;
        }
    }
}

File: Entidades/PedidoDetalle.vb
Public Class PedidoDetalle

    Private _varIdPedido As Integer
    Public Property IdPedido() As Integer
        Get
            Return _varIdPedido
        End Get
        Set(ByVal value As Integer)
            _varIdPedido = value
        End Set
    End Property

    Private _varIdProducto As Integer
    Public Property IdProducto() As Integer
        Get
            Return _varIdProducto
        End Get
        Set(ByVal value As Integer)
            _varIdProducto = value
        End Set
    End Property

    Private _varCantidad As Double
    Public Property Cantidad() As Double
        Get
            Return _varCantidad
        End Get
        Set(ByVal value As Double)
            _varCantidad = value
        End Set
    End Property

    Private _varCantidadRecibida As Double
    Public Property CantidadRecibida() As Double
        Get
            Return _varCantidadRecibida
        End Get
        Set(ByVal value As Double)
            _varCantidadRecibida = value
        End Set
    End Property

End Class
